test/integration/cms/api/sections_test.rb

```diff
@@ -109,5 +109,56 @@ def create_section
       end
     end
 
+    class SystemNameTest < ActionDispatch::IntegrationTest
+      def setup
+        @provider = FactoryBot.create(:provider_account)
+        host! @provider.external_admin_domain
+      end
+
+      test 'create without title' do
+
+        post admin_api_cms_sections_path, params: { provider_key: @provider.provider_key, format: :xml}
+
+        assert_response 422
+
+        doc = Nokogiri::XML::Document.parse(@response.body)
+        error = doc.xpath('//error').text
+
+        assert_match "Title can't be blank", error
+      end
+
+      test 'create with title but without system_name' do
+        expected_title = 'New Section'
+        expected_sysname = 'New Section'
```

Comment:
This doesn't seem like a sensible system name. Is this correct behavior? Most names that I see in actual DB are more like `new-section`

Reply:
Yes, I have a PR for that: https://github.com/3scale/porta/pull/3163

I'll have to fix this test when both are merged to master